timify-api/app.js

```diff
@@ -10,8 +10,12 @@ app.use(cors())
 app.use(express.json()); // Middleware for parsing JSON bodies from HTTP requests
 app.use(morgan())
 
-// Route to get all podcasts
 
+app.get('/', (req, res) => {
+  res.send("it works"); 
```

Comment:
do we need this?

Reply:
yes i think because i got an error message when i didn't have it.

Reply:
What kind of error message? It seems like unnecessary statement.

Reply:
it said "Cannot GET/ in the console, if i remember correctly